spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binding/BoundElementFactory.java

```diff
@@ -19,26 +19,35 @@
 import org.springframework.messaging.SubscribableChannel;
 
 /**
- * Defines methods to create/configure the {@link org.springframework.messaging.MessageChannel}s defined
- * in {@link org.springframework.cloud.stream.annotation.EnableBinding}.
+ * Defines methods to create/configure the bound elements defined by
+ * {@link org.springframework.cloud.stream.annotation.EnableBinding}.
+ *
+ * @author Marius Bogoevici
  * @author Ilayaperumal Gopinathan
  */
-public interface BindableChannelFactory {
+public interface BoundElementFactory {
```

Comment:
if you prefer 'BoundElement' that's fine, but in that case we should probably make sure to update the other occurrences of `bindingTarget` for consistency (could be a subsequent PR).